bundle/edu.gemini.pit/src/main/scala/edu/gemini/pit/ui/view/obs/ObsListModel.scala

```diff
@@ -181,15 +181,18 @@ case class ObsListModel(all:List[Observation], band:Band, gs:(ObsListGrouping[_]
 
   }
 
+  def cut(elem: ObsListElem): ObsListModel = elem match {
+    case target: ObsGroup[_] =>
+      val (_, toClear) = Seq(gs._1, gs._2, gs._3).splitAt(depth(target))
 
+      def clear(o: Observation) =
+        toClear.foldLeft(o.copy(progTime = None))((o, g) => g.clear(o))
```

Comment:
Can this be done as `_.clear(_)`? just to confuse people in the future 😄

Reply:
I would love to, but as we reverse the parameter order, I think I can't be evil.